meet-data/usapl-archive/OH-2012-03-02/meet.csv

```diff
@@ -0,0 +1,2 @@
+Federation,Date,MeetCountry,MeetState,MeetTown,MeetName
+NAPF,2012-03-02,USA,OH,Columbus,Raw Challenge
```

Comment:
The meet is named "NAPF Raw Challenge" and was previously in the NAPF folder. I think that should be unchanged.

Reply:
The qualifying documentation says that you couldn't set international records at the Raw Challenge - so despite it being titled "NAPF Raw Challange" I think it was a USAPL meet. Not really my area though, would be handy to have someone more familiar with the Arnold events chime in.